backend/src/apiserver/resource/resource_manager_util_test.go

```diff
@@ -188,7 +188,7 @@ spec:
       command:
       - python
       - -c
-      image: python:alpine3.6
+      image: python:alpine
```

Comment:
is there a reason for this image not being pinned to a version? same as on line 288

Reply:
fixed